### PR TITLE
Herbert Homolka's Student Bio

### PR DESCRIPTION
**Pull request by HerbertHomolka1**
This is my student bio together with some CI fixes.

### General Checks

-  [x] the branch is up to date with `main`/`master`
-  [x] the code works when pulled and run locally
-  [x] All CI checks pass
-  [x] all conflicts are resolved (if any)
-  [x] PR has a descriptive title
-  [x] PR has appropriate labels and milestones for easy identification
-  [x] PR it is assigned to the owner
-  [x] reviewers are assigned
-  [ ] the PR contributes only one focused change
-  [x] It is in the appropriate column in the project board (if necessary)
-  [x] has short and clear description
-  [x] is linked to an issue (if it is related)
-  [ ] feedback is addressed (if any and if it is appropriate feedback.)

### Markdown

- [x] the markdown source is formatted
- [x] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [x] all links and images work
- [x] any embedded code snippets are formatted

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "js-yaml": "^4.1.0",
     "markdownlint-cli": "^0.31.1",
     "node-fetch": "^3.2.0",
-    "prettier": "^2.3.2"
+    "prettier": "^2.8.6"
```
Comment (bermarte) [inline package.json:32]: Why did you update prettier?

Comment (HerbertHomolka1) [reply package.json:32]: Not sure, haha. I will revert it back.

Comment (HerbertHomolka1) [reply package.json]: image

Comment (samirm00) [reply package.json]: Hello, @HerbertHomolka1  Well  
1- you shouldn't change any files except students-bio and add your md file and image.
2- if you have any unknown words only in your md file you can them to .cspell.json.
3-  you should only add your changes, commit them and push them.
4- You should never change other files , especially packages.json

**Review by bermarte — APPROVED** [review]

**Pull request merged**